.gitignore

```diff
@@ -3,4 +3,5 @@
 _glooe
 mismatched_dependencies.json
 suggestions
-plugin_dependencies
\ No newline at end of file
+plugin_dependencies
+tmp
```

Comment:
why add this?

Reply:
Good point. Actually there are more entries to be deleted